islands/GlobalSearch.tsx

```diff
@@ -376,14 +395,17 @@ function ManualResult({ hierarchy, url, content }: ManualSearchResult) {
     title.push(entry);
   }
   return (
-    <a href={url} class={tw`block`}>
+    <a href={url} class={tw`flex items-center gap-4`}>
```

Comment:
can we change the background color a bit darker when mouse _hover_?

Reply:
@ije done, however the perfect hover background color for odd items is the normal background color for even items; do you think that might be a problem?